packages/www/components/Dashboard/AssetsTable/CreateAssetDialog.tsx

```diff
@@ -17,6 +17,8 @@ import { useDropzone } from "react-dropzone";
 import { Cross2Icon } from "@radix-ui/react-icons";
 import omit from "lodash.omit";
 
+const acceptedMimeTypes = [];
```

Comment:
If you're cautious about accidentally deploying this to prod, WDYT of doing it like this?
```suggestion
const acceptedMimeTypes = isStaging() ? [] : ["video/mp4"];
```

We can still revert this change later, but won't risk having this in prod.